src/javascript/JContent/ContentRoute/ContentLayout/ContentTable/reactTable/components/cells/CellStatus.jsx

```diff
@@ -12,6 +12,12 @@ export const CellStatus = ({cell, column, row}) => {
     const node = row.original;
     const lang = useSelector(state => state.lang);
     const showSubNodes = node.primaryNodeType.name !== 'jnt:page' && node.subNodes && node.subNodes.pageInfo.totalCount > 0;
+    const subNodesCountText = useMemo(() => {
+        if (showSubNodes) {
+            const count = node.subNodes.pageInfo.totalCount;
+            return count >= 99 ? '99+' : count;
```

Comment:
slight off by one error: I think we want to display 99+ if it's more than 99 (exclude the equal)